content/en/docs/armory-admin/dinghy-enable.md

```diff
@@ -58,10 +58,74 @@ hal armory dinghy enable
 {{< /tabs >}}
 
 
+## Configuring SQL
```

Comment:
WDYT about adding a `Beta` or `Early Access` badge to this section? Especially since it won't be available until 2.24 at the earliest

Reply:
Or we can switch to the PR to "draft" and add a release label so we know when to merge